test

```diff
@@ -14,6 +14,12 @@
 # COVERDIR=coverage PASSES="build_cov cov" ./test
 set -e
 
+# PASSES="functional" ./test
+if [[ ${PASSES} == *"functional"* ]]; then
```

Comment:
Getting some kind of scope error with the same script but run inside `functional_pass`:

```bash
unexpected directory layout:
	import path: github.com/coreos/pkg/capnslog
	root: /Users/gyuho/go/src
	dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/pkg/capnslog
	expand root: /Users/gyuho/go/src
	expand dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/pkg/capnslog
	separator: /
unexpected directory layout:
	import path: github.com/coreos/go-semver/semver
	root: /Users/gyuho/go/src
	dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/go-semver/semver
	expand root: /Users/gyuho/go/src
	expand dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/go-semver/semver
	separator: /
unexpected directory layout:
	import path: github.com/coreos/go-semver/semver
	root: /Users/gyuho/go/src
	dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/go-semver/semver
	expand root: /Users/gyuho/go/src
	expand dir: /Users/gyuho/go/src/github.com/coreos/etcd/cmd/vendor/github.com/coreos/go-semver/semver
	separator: /
```